src/main/java/frc/robot/Robot.java

```diff
@@ -98,6 +109,13 @@ public void teleopInit() {
   @Override
   public void teleopPeriodic() {
     SmartDashboard.putNumber("PIVOT ANGLE", RobotContainer.pivotSubsystem.getMeasurement());
+    if (RobotContainer.limeSubsystem.estimateDistance() < 3.675 && RobotContainer.feederSubsystem.isSensorTripped() && RobotContainer.limeSubsystem.estimateDistance() != 0) {
```

Comment:
make the 3.675 a constant
```suggestion
    if (RobotContainer.limeSubsystem.estimateDistance() < 3.675 && RobotContainer.feederSubsystem.isSensorTripped() && RobotContainer.limeSubsystem.estimateDistance() > 0) {
```